### ISSUE TITLE
Inherited CompositeFields don't have the Fields from the parent CompositeField one

### ISSUE THREAD

**Issue by piranna** [bikeshedder/django-composite-field#18]
I have a CompositeField with common fields, that later it's inherited by a couple of classes that add more fields. Problem is, when trying to access the fields using the Proxy, only the fields of the child classes are available, and the fields of the parent base CompositeField class are missing. If I copy the fields of the parent class to the child ones, then I can be able to access to them.

As a side note, when the fields were defined only on the parent class, they were not added to the database either.

**Comment by bikeshedder** [comment]
I have never thought of using inheritance in composite fields.

Do you have a test case showing the behavior?
Do you want to work on a PR adding support for inheritance?

**Comment by piranna** [comment]
> Do you have a test case showing the behavior?

No, I don't have it, but it would be trivial: a parent composite field `A` with an `a` field, and a child composite field `B` that inherit from `A` and have a `b` field. When creating an instance of `A`, only have the `a` field, and it's the only one in the database, and when creating an instance of `B`, both `a` and `b` fields are available both in the composite field and in the database.
